x-pack/plugins/security_solution/public/management/cypress/fixtures/artifacts_page.ts

```diff
@@ -193,8 +193,9 @@ export const getArtifactsListTestsData = (): ArtifactsFixtureType[] => [
           selector: 'fieldAutocompleteComboBox',
         },
         {
-          type: 'click',
-          customSelector: 'button[title="@timestamp"]',
+          type: 'input',
```

Comment:
Was this change intentional? How comes click on a button change into input? 🤔

Reply:
It was intentional. I'm not entirely sure why it was needed only here, but generally speaking, we prefer using keyboard navigation when selecting combobox items.

Previously, it was a click on an option with a specified name; now, it's typing the option name into the search field and selecting the first option, which is the exact match. This way should be more reliable no matter what order the items are displayed in, too.